cmd/users.go

```diff
@@ -287,25 +288,25 @@ var cmdDeploymentEnableTotp = &cobra.Command{
 		var totpInfo common.TotpResponse
 		if err = json.Unmarshal(body, &totpInfo); err != nil {
 			log.Fatal(err)
-			return
 		}
 
-		qrBuilder := qr.New()
-		qrBuilder.Get(fmt.Sprintf("otpauth://totp/%s?secret=%s&issuer=Inertia",
+		// Display QR code so users can easily add their keys to their
+		// authenticator apps
+		qr.New().Get(fmt.Sprintf("otpauth://totp/%s?secret=%s&issuer=Inertia",
```

Comment:
😍

Reply:
Yea... amazingly this actually works with Google Authenticator. It probably works with other apps too.

Reply:
thats sooooo slick